lib/game_team.rb

```diff
@@ -14,6 +14,12 @@ def self.game_teams=(value)
     @@game_teams = value
   end
 
+  def self.season_games(games)
```

Comment:
Am I correct in assuming the parameter `games` is a collection of all games you want to find matching gameteam entries for?

Reply:
It's similar to your methods that you showed on slack. The `Game - games_in_a_season` return an array of game objects that have a similar season as the parameter. `GameTeam - season_games` takes the return value of `games_in_a_season` and through `GameTeam.all` for matching `game_ids`